docs/WASI-overview.md

```diff
@@ -1,163 +1,9 @@
-of a libpreopen-like layer and a system call wrapper layer (derived
-from the "bottom half" of
-[cloudlibc](https://github.com/NuxiNL/cloudlibc)).
-The system call wrapper layer makes calls to the actual WASI
-implementation, which may map these calls to whatever the
-surrounding environment provides, whether it's native OS resources,
-JS runtime resources, or something else entirely.
-
-[This libc is part of a "sysroot"](https://github.com/WebAssembly/reference-sysroot),
-which is a directory containing compiled libraries and C/C++ header
-files providing standard library and related facilities laid out in
-a standard way to allow compilers to use it directly.
-
-With the [LLVM 8.0](http://llvm.org/)
-release, the WebAssembly backend is now officially stable, but LLVM
-itself doesn't provide a libc - a standard C library, which you
-need to build anything with clang. This is what the WASI-enabled
-sysroot provides, so the combination of clang in LLVM 8.0 and the
-new WASI-enabled sysroot provides usable Rust and C compilation
-environments that can produce wasm modules that can be run in
-[Wasmtime] with WASI support, in browsers with the WASI polyfill,
-and in the future other engines as well.
-
-![WASI software architecture diagram](wasi-software-architecture.png "WASI software architecture diagram")
-
-## Future Evolution
-
-The first version of WASI is relatively simple, small, and
-POSIX-like in order to make it easy for implementers to prototype
-it and port existing code to it, making it a good way to start
-building momentum and allow us to start getting feedback based on
-experience.
-
-Future versions will change based on experience
-and feedback with the first version, and add features to address
-new use cases. They may also see significant architectural
-changes. One possibility is that this API could
-evolve into something like
-[Fuchsia](https://en.wikipedia.org/wiki/Google_Fuchsia)'s
-low-level APIs, which are more complex and abstract, though also
-more capable.
-
-We also expect that whatever WASI evolves into in the future, it
-should be possible to implement this initial API as a library
-on top.
-
-## Can WASI apps run on the Web?
-
-Yes! We have a polyfill which implements WASI and runs in browsers.
-At the WebAssembly level, WASI is just a set of callable functions that
-can be imported by a .wasm module, and these imports can be implemented
-in a variety of ways, including by a JavaScript polyfill library running
-within browsers.
-
-And in the future, it's possible that
-[builtin modules](https://github.com/tc39/ecma262/issues/395)
-could take these ideas even further allowing easier and tighter
-integration between .wasm modules importing WASI and the Web.
-
-## Work in Progress
-
-WASI is currently experimental. Feedback is welcome!
-
-[Wasmtime]: https://github.com/bytecodealliance/wasmtime
+[WASI Subgroup]: https://github.com/WebAssembly/wasi
+[wasi.dev]: https://wasi.dev/interfaces
```

Comment:
I think this link name should be `[Interfaces page on wasi.dev]` to match the reference above?

Reply:
Yep, fixed.